### PR TITLE
add custom plugin author popouts

### PR DESCRIPTION
**Pull request by Vendicated**
github and website are obtained from discord connections

image

https://github.com/Vendicated/Vencord/assets/45497981/3fc212ec-18ca-48c1-a0d2-588852d53285

### CODE REVIEW

**Comment by Vendicated** [comment]
also fixes the infamous plugin devs showing up as BOT bug

File: src/components/PluginSettings/ContributorModal.tsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+            .filter(p => !p.name.endsWith("API"))
+            .sort((a, b) => Number(a.required ?? false) - Number(b.required ?? false));
+    }, [user.id, user.username]);
+
+    return (
+        <>
+            <div className={cl("header")}>
+                <img
+                    className={cl("avatar")}
+                    src={user.getAvatarURL(void 0, 512, true)}
```
Comment (TheKodeToad) [inline src/components/PluginSettings/ContributorModal.tsx:80]:
```suggestion
                    src={user.getAvatarURL(undefined, 512, true)}
```

was this an accident..?

File: src/components/PluginSettings/PluginModal.tsx [hunk 2]
```diff
@@ -214,6 +202,19 @@ export default function PluginModal({ plugin, onRestartNeeded, onClose, transiti
...
                             showUserPopout
                             renderMoreUsers={renderMoreUsers}
+                            renderUser={(user: User) => (
+                                <Clickable
+                                    className={AvatarStyles.clickableAvatar}
+                                    onClick={() => openContributorModal(user)}
+                                >
+                                    <img
+                                        className={AvatarStyles.avatar}
+                                        src={user.getAvatarURL(void 0, 80, true)}
```
Comment (TheKodeToad) [inline src/components/PluginSettings/PluginModal.tsx:212]:
```suggestion
                                        src={user.getAvatarURL(undefined, 80, true)}
```

**Review by TheKodeToad — APPROVED** [review]

File: src/components/PluginSettings/ContributorModal.tsx (hunk 1 above)
Comment (Vendicated) [reply src/components/PluginSettings/ContributorModal.tsx:80]: void 0 = undefined

**Pull request merged**